가짜 토스 페이먼츠 서버 실행 문의

## Question

https://github.com/samchon/fake-toss-payments-server/blob/2efc253d4420615f49e7a5b179fabb6903566736/README.md?plain=1#L108

README.md 내용대로 해당 저장소를 복제하여 모듈 인스톨, 빌드 후 실행 하려하니 아래와 같이 에러가 발생합니다

```shell
$ npm run start

> fake-toss-payments-server@1.0.2 start
> pm2 start lib/executable/server.js -i 1 --name fake-toss-payments-server --wait-ready --listen-timeout 120000 --kill-timeout 15000 --

[PM2][ERROR] File ecosystem.config.js not found
```

별도로 src\FakeTossConfiguration.ts 파일에서 수정한 내용은 없는데, 추가 설정이 필요한 걸까요? 관련 파일에 대한 정보를 찾지 못하여 질문 남깁니다.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "build:swagger": "nestia swagger",
     "dev": "npm run build -- --watch",
     "package:api": "npm run build:api && cd packages/api && npm publish",
-    "start": "pm2 start lib/executable/server.js -i 1 --name fake-toss-payments-server --wait-ready --listen-timeout 120000 --kill-timeout 15000 --",
+    "start": "pm2 start lib/executable/server.js -i 1 --name fake-toss-payments-server --wait-ready --listen-timeout 120000 --kill-timeout 15000",
     "start:reload": "pm2 reload fake-toss-payments-server",
     "stop": "pm2 delete fake-toss-payments-server",
     "test": "node lib/test",

--- a/src/test/features/examples/test_fake_billing_payment.ts
+++ b/src/test/features/examples/test_fake_billing_payment.ts
@@ -24,7 +24,6 @@ export async function test_fake_billing_payment(): Promise<void> {
             },
         );
     assertType<ITossBilling>(billing);
-    console.log(TestConnection.FAKE.headers);
 
     // 간편 결제 카드로 결제하기
     const payment: ITossPayment = await toss.functional.v1.billing.pay(

